sdk/communication/AzureCommunicationChat/AzureCommunicationChat.xcodeproj/project.pbxproj

```diff
@@ -508,6 +514,7 @@
 				0A9207EC26839B3100469608 /* Recordings */,
 				D1CD93AB258D6C0300409613 /* Util */,
 				D1B7EAFC257F03A1004F384A /* Info.plist */,
+				D1E4DD8626B1D01600B7691E /* test-settings.plist */,
```

Comment:
Same here. `test-settings.plist` isn't committed to the repo (purposefully) so this should be removed.